### PR TITLE
WIP: Replace fetching of remote torrents with the usage of local ones

### PR DESCRIPTION
**Pull request by devos50**
Very dangerous to be depending on the internet since our used torrent is not working anymore: http://torrent.fedoraproject.org/torrents/Fedora-20-i386-DVD.torrent.

### CODE REVIEW

**Comment by lfdversluis** [comment]
Is this test testing downloading from a http link or doesn't it matter what the source is?

File: Tribler/Test/test_gui_dialogs.py [hunk 1]
```diff
@@ -96,7 +96,7 @@ def do_save_dialog():
             self.assert_(isinstance(dialog, AddTorrent), 'could not find AddTorrent')
 
             self.callLater(1, lambda: do_assert(dialog))
-            dialog.magnet.SetValue(r'http://torrent.fedoraproject.org/torrents/Fedora-20-i386-DVD.torrent')
```
Comment (whirm) [inline Tribler/Test/test_gui_dialogs.py]: Use the constants defined in Tribler/Test/test_libtorrent_download.py for torrent paths, so we can change it easily if we need to.

**Comment by lfdversluis** [comment]
btw @whirm this is why currently 5 other tests are failing. The saveas in my PR is caused by this (because it failed to download there is nothing to save, nothing to resume, nothing to delete etc.)

File: Tribler/Test/test_gui_dialogs.py (hunk 1 above)
Comment (devos50) [inline Tribler/Test/test_gui_dialogs.py]: Ah didn't know that one, thanks!

**Comment by devos50** [comment]
Indeed, I'm currently fixing all these instances where torrents are fetched from the internet (and is not necessarily needed). Will add it to this PR so please, do not merge it yet.

**Comment by devos50** [comment]
retest this please

**Comment by devos50** [comment]
@whirm I'm done this this PR. I will hunt the bug where the video will not play next and create a new PR for this.

**Comment by whirm** [comment]
@devos50 good

File: Tribler/Test/test_libtorrent_download.py [hunk 2]
```diff
@@ -9,12 +9,12 @@
...
 
-TORRENT_R = r'http://torrent.fedoraproject.org/torrents/Fedora-Live-Workstation-x86_64-21.torrent'
-TORRENT_INFOHASH = binascii.unhexlify('89f0835dc2def218ec4bac73da6be6b8c20534ea')
+TORRENT_R = r'http://torrent.ubuntu.com/releases/wily/release/server/ubuntu-15.10-server-ppc64el.iso.torrent'
+TORRENT_INFOHASH = binascii.unhexlify('9874df3783f8de52f944032b9affe4a5c88bc9fe')
 TORRENT_FILE = os.path.join(TESTS_DATA_DIR, "ubuntu-15.04-desktop-amd64.iso.torrent")
 TORRENT_FILE_INFOHASH = binascii.unhexlify("fc8a15a2faf2734dbb1dc5f7afdc5c9beaeb1f59")
 TORRENT_VIDEO_FILE = os.path.join(TESTS_DATA_DIR, "Night.Of.The.Living.Dead_1080p_archive.torrent")
-TORRENT_VIDEO_FILE_INFOHASH = binascii.unhexlify("0d6bbc31db060e7082f85a87194d4273aea9924c")
+TORRENT_VIDEO_FILE_INFOHASH = binascii.unhexlify("90ed3962785c52a774e89706fb4f811a468e6c05")
```
Comment (whirm) [inline Tribler/Test/test_libtorrent_download.py]: What is that infohash? If I didn't mix things up, this was the infohash of the night of the living dead torrent.

Comment (devos50) [inline Tribler/Test/test_libtorrent_download.py]: That old info hash (`0d6bbc31db060e7082f85a87194d4273aea9924c`) was wrong and didn't correpond to Night of the living dead. I found this out when the test was hanging. After I determined the right info hash and replace it, it started to find the torrent in the list of downloads.

File: Tribler/Test/test_my_channel.py [hunk 3]
```diff
@@ -31,7 +32,7 @@ def added_rss():
                 60, lambda: len(managefiles.GetItems()) > 0, do_files_check, 'Channel did not have torrents')
 
         def do_rss():
-            self.managechannel.rss_url.SetValue(r'http://torrent.fedoraproject.org/rss20.xml')
+            self.managechannel.rss_url.SetValue(os.path.join(TESTS_DATA_DIR, 'test_rss.xml'))
```
Comment (whirm) [inline Tribler/Test/test_my_channel.py]: maybe we should test both local and remote URIS? I guess we could create a local http server that serves that same xml?

Comment (lfdversluis) [inline Tribler/Test/test_my_channel.py]: We could add additional tests adding these in a separate PR ?

Comment (whirm) [inline Tribler/Test/test_my_channel.py]: Sure, create some issue so we don't forget.

File: Tribler/Test/test_libtorrent_download.py (hunk 2 above)
Comment (whirm) [inline Tribler/Test/test_libtorrent_download.py]: Oh :( my bad

**Pull request merged**